gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart

```diff
@@ -7,15 +7,59 @@ import 'package:ubuntu_service/ubuntu_service.dart';
 import 'package:ubuntupro/core/agent_api_client.dart';
 import 'package:ubuntupro/pages/subscription_status/subscription_status_model.dart';
 import 'package:ubuntupro/pages/subscription_status/subscription_status_page.dart';
+import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
 import 'package:wizard_router/wizard_router.dart';
 
 import '../../utils/build_multiprovider_app.dart';
+import '../../utils/url_launcher_mock.dart';
 
 void main() {
   group('subscription info:', () {
     final client = FakeAgentApiClient();
     registerServiceInstance<AgentApiClient>(client);
     final info = SubscriptionInfo();
+
+    group('footer links:', () {
+      final landscape = LandscapeSource()..ensureOrganization();
+      testWidgets('user', (tester) async {
```

Comment:
Can we have a similar test case that finds no footer links for the org case?

Reply:
I have `expect(find.text(lang.ubuntuProManage), findsNothing);` in the other org test cases, I figured that was enough?

Reply:
Yeap, that may suffice indeed. But if we go with your suggestion for the org case it'd be good to have it more specific.

Reply:
Now that we're sure we won't get that copy, let's call the day :)